Xamarin.Forms.Platform.iOS/Renderers/PhoneMasterDetailRenderer.cs

```diff
@@ -391,7 +391,11 @@ void UpdatePanGesture()
 				return;
 			}
 
-			UITouchEventArgs shouldReceive = (g, t) => !(t.View is UISlider);
+   			bool shouldReceive(UIGestureRecognizer g, UITouch t)
```

Comment:
Is this not an issue on iPad?

Reply:
We avoid to received the touch in the gesture (MasterDetailPage) if the View is and SwipeView  (or and Slider) on Phone, but is not necessary on the Tablet renderer (because is open).